src/foam/nanos/auth/Authorizer.java

```diff
@@ -18,4 +18,7 @@
   void authorizeOnRead(X x, FObject obj) throws AuthorizationException;
   void authorizeOnUpdate(X x, FObject oldObj, FObject newObj) throws AuthorizationException;
   void authorizeOnDelete(X x, FObject obj) throws AuthorizationException;
+  boolean checkGlobalRead(X x);
+  boolean checkGlobalRemove(X x);
+  String getPermissionPrefix();
```

Comment:
This isn't applicable to all `Authorizer` implementations so I don't think it should be part of the interface.